deploy/bdr.registries.notifications.devel/app.env

```diff
@@ -17,4 +17,9 @@ BDR_SERVER_URL=http://localhost/
 
 USE_ZOPE_LAYOUT=True
 
-EMAIL_SENDER=
\ No newline at end of file
+EMAIL_SENDER=
+
+FGASESREGISTRY_URL=http://172.19.0.1:5000
```

Comment:
Remove FGASESREGISTRY_URL value. This value depends on the machine, the network you are connected to etc.